core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -69,8 +69,9 @@
   private static final String REGEX_STRING = "(\\$\\{([^{}]*)\\})";
   /** Regex to find ${key} for variable substitution. */
   private static final Pattern CONF_REGEX = Pattern.compile(REGEX_STRING);
-  /** Set of properties. */
-  private static final Properties PROPERTIES = new Properties();
+  /** Map of properties. */
+  private static final ConcurrentHashMapV8<String, String> PROPERTIES_MAP =
```

Comment:
NIT: I think it is fine to just call this `PROPERTIES` to be concise. But I am netural on the name of this map here, leave it to you.